R/estimate_dispersion.R

```diff
@@ -13,6 +13,13 @@
 #'    list are paired with elements of `pt_nowcast_mat_list`.
 #' @param n Integer indicating the number of reporting matrices to use to
 #'    estimate the dispersion parameters.
+#' @param fun_to_aggregate Function that will  will operate along the nowcast
+#'    vectors after summing across delays. Eventually, we can add things like
+#'    mean, but for now since we are only providing a negative binomial
+#'    observation model, we can only allow sum (min or max would work but
+#'    wouldn't make much sense).
+#' @param k Integer indicating the number `width` of the call to `rollapply`.
```

Comment:
Noting this is not a very useful piece of documentation

Reply:
Took another pass...